server/index.js

```diff
@@ -22,7 +24,11 @@ app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
 app.use(methodOverride());
-app.use(express.static(`${__dirname}/public`));
+app.use(express.static(path.join(__dirname, 'public')));
+
+
+app.use(passport.initialize());
```

Comment:
Hi! clinton and chima. well done on this output. I see a duplicate of passport.initialize, do you mind checking to correct it